server/src/main/java/org/solarbank/server/dto/UserInputDto.java

```diff
@@ -0,0 +1,24 @@
+package org.solarbank.server.dto;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class UserInputDto {
```

Comment:
It's convention to call these request objects something like `CalculateRequest` as it represents the incoming request from the client.

Reply:
renamed to CalculateRequest